python/graphstorm/sagemaker/utils.py

```diff
@@ -225,6 +226,8 @@ def download_graph(graph_data_s3, graph_name, part_id, local_path, sagemaker_ses
         Path to store graph data
     sagemaker_session: sagemaker.session.Session
         sagemaker_session to run download
+    node_mapping_prefix_s3: str, optional
```

Comment:
```suggestion
    node_remapping_prefix_s3: str, optional
```
Let's call it remapping.